config/locales/support_interface/personas.yml

```diff
@@ -0,0 +1,34 @@
+en:
+  personas:
+    permissions:
+      admin: This user has both manage_organisations and manage_users permissions.
+      user: This user can only view applications.
+    organisation_membership_explanation:
+      self_ratified: This user belongs to a single provider, which has no relationships with other providers.
+      multiple: This user belongs to multiple providers.
+      no_courses: This user belongs to a single provider which has no open courses.
+    users:
```

Comment:
I've moved some of the persona user information to the locales. It might be nice to drive the creation service using a separate YML file, but we can do this in another PR if we want to I think

Reply:
Yea I think we are currently using the locales to drive the list which isn't ideal in the page, and if we move them to a dedicated file then we can have our source of truth and as you mentioned use that list for creation. But happy for that to come later

Reply:
Discussed that we'll raise a card for this and pick it up in a sprint soon